SpatialGDK/Source/SpatialGDK/Public/Interop/ActorSystem.h

```diff
@@ -206,6 +207,9 @@ class ActorSystem
 
 	TSet<Worker_EntityId_Key> PresentEntities;
 
+	// Only the ones we know the EntityId for beforehand.
```

Comment:
When isn't this the case?

Reply:
Sorry, removed. Leftover from my early ignorance.